tooling/github-configuration.md

```diff
@@ -0,0 +1,103 @@
+# GitHub InnerSource Configuration
+
+- [Enterprise Setting](#enterprise-setting)
+  - [Repository policies](#repository-policies)
+    - [Base permissions](#base-permissions)
+    - [Repository creation](#repository-creation)
+    - [Repository forking (Private / Internal)](#repository-forking-private--internal)
+- [Organization Setting](#organization-setting)
+  - [Member privileges](#member-privileges)
+    - [Base permissions](#base-permissions-1)
+    - [Repository creation](#repository-creation-1)
+    - [Repository forking](#repository-forking)
+- [GitHub Enterprise Server Setting](#github-enterprise-server-setting)
+  - [GitHub Connect - Server statistics](#github-connect---server-statistics)
+- [Resources](#resources)
+
+## Enterprise Setting
+
+### Repository policies
+
+#### Base permissions
```

Comment:
From @JustinGosses  ([The original review comment](https://docs.google.com/document/d/1lPR90dIiWKlsNlV46xhGkR9pP0GNYmeAB7xOk-_MAYo/edit?disco=AAAA1tS7-bE))

This section is a bit confusing to me. Are the first two org-level policies for what people can set up as repo access policies and the last three repo permissions policies? If so, those groups should maybe be in separate tables. The repository permissions should have 5 roles now right? [https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization](https://www.google.com/url?q=https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization&sa=D&source=docs&ust=1691047026256627&usg=AOvVaw2I-V0yK8XPzUWB7hi5sTnB)